app/scripts/lib/createDupReqFilterMiddleware.test.js

```diff
@@ -0,0 +1,25 @@
+import createDupReqFilterMiddleware from './createDupReqFilterMiddleware';
+
+describe('createDupReqFilterMiddleware', () => {
+  it('call function next if request is seen first time', () => {
+    const filterFn = createDupReqFilterMiddleware();
+    const request = { id: 1 };
+    const nextMock = jest.fn();
+    const endMock = jest.fn();
+    filterFn(request, undefined, nextMock, endMock);
```

Comment:
```suggestion

    filterFn(request, undefined, nextMock, endMock);

```
echoing what @Gudahtt mentioned in a [comment in another PR](https://github.com/MetaMask/KeyringController/pull/169/files#r1035969577)

> Nit: typically for tests, we use newlines to separate the different phases of the test: arrange, act, and assert. Arrange is the setup, act is where we do the thing that is under test, and assert is where we check that it works.